corehq/ex-submodules/phonelog/migrations/0005_add_forceclose_entry_20160408_1530.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('phonelog', '0004_merge'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ForceCloseEntry',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('domain', models.CharField(max_length=100, db_index=True)),
```

Comment:
yes, its an extra index that we don't need. Basically every time to create a text / varchar column with an index Django will create 2 indexes (one for exact matches and one for partial matches). If you look at your local DB after running the migration you'll see an index called `phonelog_forcecloseentry_domain_542fcaceb2264695_like` which will never get used.

There are two ways to get rid of it:
1. add the index in a separate operation:
```
operations = [
    migrations.CreateModel(
            name='ForceCloseEntry',
            fields=[
                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
                ('domain', models.CharField(max_length=100)), # no index
        ...], ...),
    migrations.AlterField(
            model_name='ForceCloseEntry',
            name='domain',
            field=models.CharField(max_length=100, db_index=True),  # add the index here
            preserve_default=True,
        ),
```

2. drop the index after it's been created
```
HqRunSQL(
            'DROP INDEX IF EXISTS form_processor_caseattach_attachment_uuid_4c1d2c3ea75567cc_like',
            NOOP_REVERSE
        ),
```